san/pandas_utils.py

```diff
@@ -2,7 +2,7 @@
 
 
 def convert_to_datetime_idx_df(data):
-    df = pd.DataFrame(data)
+    df = pd.DataFrame(data, columns=data[0].keys())
```

Comment:
Does that change anything for this result and also for other queries results or it is the default one?

Reply:
This is needed for the news metric, because otherwise, when the dataframe is being made from the "transformed" query, it turns it back to alphabetical order, which is not what we want for the news metric. The ordinary metrics are okay with this, but I you are right in the sense, that the transformed queries are ordered differently. I've just updated the eth_top_transactions transform function, so that it also reorders the result there.

Reply:
If this column reordering isn't done, then the DataFrame will be automatically set to be in alphabetical order

Reply:
for news you want them to be the same as they are in the graphql query request ?
```
        datetime,
        title,
        description,
        sourceName,
        url
```

Reply:
I think this is feature of python 3.6+: https://docs.python.org/3.6/whatsnew/3.6.html#whatsnew36-compactdict

Reply:
By default it is description->sourceName->title, I wanted it reordered

Reply:
I think it can't be depended upon (since lower versions user won't get the same order) , but also doesn't hurt

Reply:
won't it result in a runtime error if you execute it with python < 3.6?

Reply:
it will work, just the order of keys won't be preserved in lower versions